src/patterns/ec2-app/base.ts

```diff
@@ -377,6 +449,115 @@ export class GuEc2App extends Construct {
       }
     }
 
+    if (props.googleAuth?.enabled) {
+      const prefix = `/${scope.stage}/${scope.stack}/${app}`;
```

Comment:
I feel like this is probably already defined in CDK somewhere. @akash1810 any thoughts? Or if not, perhaps we should define it and make it easy to use both internally and externally.

Reply:
I don't think it is centrally defined. The closest I can think of is within [the SSM policy](https://github.com/guardian/cdk/blob/0110e9786aa0d1db49a6569a18eb082585dd8eef/src/constructs/iam/policies/parameter-store-read.ts#L15). 👍🏽 to DRYing it out.

Reply:
(Will do in a follow-up as feels like a general thing.)